src/renderer/pages/Dashboard.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  useGetIssuedCarbonByMethodologyQuery,
+  useGetIssuedCarbonByProjectTypeQuery,
+  useGetProjectsCountQuery,
+} from '@/api';
+import { IndeterminateProgressOverlay, ProjectsView, SkeletonProjectsView, Tabs, UnitsView } from '@/components';
+import { useQueryParamState } from '@/hooks';
+import { ProjectsStatusCount } from '@/schemas/ProjectsStatusCount.schema';
+import { ProjectsHostedCount } from '@/schemas/ProjectsHostedCount.schema';
+
+const Dashboard: React.FC = () => {
+  const [status /* set func here */] = useQueryParamState('status', 'true');
+  const [hostRegistry /* set func here */] = useQueryParamState('hostRegistry', 'true');
+
+  const {
+    data: projectsStatusCountData,
```

Comment:
lets have each chart manage its api hooks. same goes for the counts at the top.

for example, there are currently 6 `<Card>` tags on the dashboard layout, presenting various types of data. each one should get its data independently from all of the others and be completely self contained